samples/files.js

```diff
@@ -312,6 +313,79 @@ async function generateSignedUrl(bucketName, filename) {
   // [END storage_generate_signed_url]
 }
 
+async function generateV4ReadSignedUrl(bucketName, filename) {
+  // [START storage_generate_signed_url_v4]
+  // Imports the Google Cloud client library
+  const {Storage} = require('@google-cloud/storage');
+
+  // Creates a client
+  const storage = new Storage();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const bucketName = 'Name of a bucket, e.g. my-bucket';
+  // const filename = 'File to access, e.g. file.txt';
+
+  // These options will allow temporary read access to the file
+  const options = {
+    version: 'v4',
+    action: 'read',
+    expires: Date.now() + 1000 * 60 * 60, // one hour
```

Comment:
Missed this in the last review, please change expires time to 15 minutes to be consistent with other language samples. For Read and Upload samples.